.github/workflows/CI.yml

```diff
@@ -308,7 +308,7 @@ jobs:
       - name: Install Yarn Latest
         if: matrix.node-version == 22
         run: |
-          npm i -g yarn@latest
+          npm i -g @yarnpkg/cli-dist@latest
```

Comment:
Could you separate any change other than the version bump?

Reply:
Yeah. Moving this back to draft while I figure out what all is needed to make the smoke tests pass then I'll split it up as needed.